build.gradle

```diff
@@ -876,3 +876,9 @@ artifactory {
         }
     }
 }
+
+wrapper {
+    gradleVersion = '6.9.2'
+    distributionType = Wrapper.DistributionType.BIN
+    distributionUrl="https://gradleproxy:gradleproxy@software.r3.com/artifactory/gradle-proxy/gradle-$gradleVersion-bin.zip"
```

Comment:
This URL contains an artifactory username and password. Please can we make this work without sharing a user account?

Reply:
We cache Gradle distributions in Artifactory so that we don't spam the Gradle servers (which will rate-limit us if we make too many requests). This URL is our standard approach across Corda 4, Corda 5 and other repositories as devised by the Build Team, and so there is no reason for Quasar to behave any differently.